### ISSUE TITLE
Add prefetch steep to download all remote resources on begin of docs build

### ISSUE THREAD

**Issue by Czaki** [napari/docs#1089]
## 🧰 Task

We currently have multiple failures of docs build in the middle of a long run because of the inaccessibility of some remote resources. 

As far as I checked, all downloads are done in a cacheable way (pooch or other lib like this).

So we could add a prefetch steep befor start building html. 

Advantages:
1. Early fail on inaccessible resources, no more parsing of long logs 
2. Better ability to cache it. 
3. If data are too big on cache, we could put them on our server, and prepopulate before tests with source codes pointing to original data. 

Disadvantages
1. Next list to keep in sychronisation
